### ISSUE TITLE
ARIA attribute unsupported or prohibited in logo bar on institute homepage

### ISSUE THREAD

**Issue by poojagunturu96** [middlebury/midd-frontend#771]
Siteimprove report: https://my2.siteimprove.com/Inspector/7333/A11Y/Page?pageId=64766853479&impmd=D97548790CE293BDC08F6D9B84D3E607&conf=a+aa+aria+si&issue=cantTell+failed&wcag=twopointone&decision=dismissed+falsePositive+none#/sia-r18/failed/VE5VcUVRU2M=;NTE5M2FjYzA=
